src/drivers/block_dev/block_dev.c

```diff
@@ -29,18 +25,10 @@
 
 #include <embox/block_dev.h>
 
-#define MAX_DEV_QUANTITY OPTION_GET(NUMBER,dev_quantity)
+#define MAX_DEV_QUANTITY 16 /* XXX */
```

Comment:
Oh, sorry. I see that this constant is not even used in block_dev.c, so it could be removed and replaced with option_get. Will fix it.